src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -152,39 +170,84 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
                     <Typography variant="h2" gutterBottom>
                       {t('registration.files_and_license.files')}
                     </Typography>
+
                     {files.length > 0 && (
                       <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', mb: '2rem' }}>
-                        {files.map((file) => {
-                          const associatedFileIndex = associatedArtifacts.findIndex((artifact) => {
-                            if (associatedArtifactIsFile(artifact)) {
-                              const associatedFile = artifact as AssociatedFile;
-                              return associatedFile.identifier === file.identifier;
-                            }
-                            return false;
-                          });
+                        <TableContainer component={Paper}>
+                          <Table sx={alternatingTableRowColor}>
+                            <TableHead>
+                              <TableRow>
+                                <TableCell>{t('common.name')}</TableCell>
+                                <TableCell></TableCell>
```

Comment:
Her vil Axe klage på at det er en kolonne uten heading. Tror vi kan fjerne den, og legge slette-knapp i samme kolonne som navnet
```suggestion
```

Reply:
For å få til samme innhold i raden under må kanskje TableCell ha noe sånt
```
<Box sx={{display:"flex", gap:"0.5rem", justifyContent:"space-between">
   <Typography>...
   <IconButton>...
```